frontend/src/app/game/game/game.component.ts

```diff
@@ -30,10 +30,11 @@ export class GameComponent {
 
     // send current game result into outro
     gameService.gameState$.pipe(
-      map(state => state.stats),
-      map(stats => ({
-        dead: stats.deaths.total,
-        cost: stats.costs.total,
+      filter(states => states.length > 0),
+      map(states => states[states.length - 1]),
```

Comment:
For reset result when game restarted could be better
`map(states => states?.length ? states[states.length - 1] : undefined),`

Next `map` must be edited to correctly process undefined.